dev-docs/modules/topicsFpdModule.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page_v2
+page_type: module
+title: Module - topicsFpdModule
+description: how to get topics data using Topics API
+module_code : topicsFpdModule
+display_name : Topics FPD Module
+enable_download : true
+sidebarType : 1
+---
+
+# Overview
+This module is use to call the Topics API (document.browsingTopics()) which will fetch the first party domain as well third party domain(Iframe) topics data which will be sent onto user.data in bid stream. 
+
+
+To learn more about topics in general, go [here](https://blog.google/products/chrome/get-know-new-topics-api-privacy-sandbox/).
```

Comment:
```suggestion
To learn more about topics in general, visit [googles blog "Get to know the new Topics API for Privacy Sandbox"](https://blog.google/products/chrome/get-know-new-topics-api-privacy-sandbox/).
```

I personally find links that are just marked as "here" harder to find and less intuitive. WDYT?

Reply:
Done